packages/theme/src/components/slider.ts

```diff
@@ -15,13 +15,19 @@ function thumbOrientation(props: StyleFunctionProps): SystemStyleObject {
       transform: `translateX(-50%)`,
       _active: {
         transform: `translateX(-50%) scale(1.15)`,
+        _disabled: {
+          transform: `translateX(-50%)`,
+        },
```

Comment:
Do we really need these styles?

Reply:
Please see below comment, same discussion applies but onto the x-axis attributes.